internal/pkg/print/markdown/content/testdata/content-WithAggregateTypeDefaults.golden

```diff
@@ -0,0 +1,116 @@
+Usage:
+
+module "foo" {
+  source = "github.com/foo/bar"
+
+  id   = "1234567890"
+  name = "baz"
+
+  zones = ["us-east-1", "us-west-1"]
+
+  tags = {
+    Name         = "baz"
+    Created-By   = "first.last@email.com"
+    Date-Created = "20180101"
+  }
+}
+
+## Inputs
+
+These variables are defined:
+
+### unquoted
+
+Description: -
```

Comment:
I personally like to still show `Description: -` (or `Description: not available` or similar), to make the blocks consistent. Otherwise some block would have description some wouldn't. And also we cannot remove empty description from  `table` or `json` format, so I think we should keep it for `document` type as well.

Reply:
any thought on this?

Reply:
I see a couple of options here:
1) "Description: This is about..." and "Description: -"
2) "Description: This is about..." and "Description: n/a"
3) "This is about..." and "" (nothing)
Which one is your favorite? I'll leave this up to you.

Reply:
I personally like option 2, both for markdown `document` and `table` (specially for consistency). I also believe this should remain as is in `json` because of potential integration might have been setup on it already and also having an _empty_ item in a json object is perfectly normal.

Reply:
Agreed. For the sake of keeping this feature branch lean, why not go with option 1 for consistency and then provide another update that provides 2) across all markdown implementations?

Reply:
Makes sense, I will do option 1 here and open a PR for option 2 later.

Reply:
I'm gonna resolve this since the feature currently works as is.